guides/common/assembly_preparing-provisioning-content.adoc

```diff
@@ -1,4 +1,6 @@
+ifdef::katello,orcharhino,satellite[]
 include::modules/con_preparing-provisioning-content.adoc[]
+endif::[]
 
 include::modules/proc_preparing-a-synchronized-kickstart-repository.adoc[leveloffset=+1]
```

Comment:
```suggestion
include::modules/con_preparing-provisioning-content.adoc[]

ifdef::katello,orcharhino,satellite[]
include::modules/proc_preparing-a-synchronized-kickstart-repository.adoc[leveloffset=+1]
endif::[]
```

With that, we'd also have to hide the link to "Managing content" in `guides/common/modules/con_preparing-provisioning-content.adoc`.